### ISSUE TITLE
Ship Mock in pypi.org

### ISSUE THREAD

**Issue by praiskup** [rpm-software-management/mock#1148]
Having Mock distributed PyPI seems like a way to make RPM builds trivial
on non-RPM systems (RPM distributions typically already have pre-packaged
mock-as-RPM already).

Especially with the `--use-bootstrap-image`  feature that is about to be
by default ON soon (#1101) this is really a low-hanging fruit.

**Comment by Conan-Kudo** [comment]
We still depend on a container runtime being installed on the host, particularly Podman. That makes it being in PyPI less useful than you'd ordinarily think.

**Comment by praiskup** [comment]
That's right. And when you already have Podman runtime, you can run Mock-in-Fedora-container.  But we still should do this (especially if we plan to wrap some parts of the Mock code into a library).

**Comment by Conan-Kudo** [comment]
Fair. So do we have `mockbuild` locked in already?

**Comment by praiskup** [comment]
Done: https://pypi.org/project/mockbuild/0.0/

**Comment by thebeanogamer** [comment]
Just to add to this, having `mockbuild` on PyPi would be really useful for [FedoraReview](https://pagure.io/FedoraReview). You have to do some really unpleasant hacks right now to be able to develop in a `venv` because `mockbuild` is only available on the system Python install.

**Comment by praiskup** [comment]
FTR, after a few months I received this e-mail from PyPI

> Your new organization named 'rpm-software-management' has been approved and
> created.
> You are the owner of the new organization and can log in to manage
> organization projects, collaborators, teams, or settings.

**Comment by praiskup** [comment]
What part do you depend on?

**Comment by mkoncek** [comment]
> What part do you depend on?

basically only `mockbuild.config.simple_load_config`.
